roxctl/common/tls.go

```diff
@@ -88,7 +87,7 @@ func tlsConfigOptsForCentral() (*clientconn.TLSConfigOptions, error) {
 			return nil, errors.Errorf("CA certificates file %s contains no certificates!", flags.CAFile())
 		}
 		if flags.SkipTLSValidation() != nil && *flags.SkipTLSValidation() {
-			fmt.Fprintln(os.Stderr, "Warning: --insecure-skip-tls-verify has no effect when --ca is set")
+			CLIEnvironmentSingleton().Logger().WarnfLn("Warning: --insecure-skip-tls-verify has no effect when --ca is set")
```

Comment:
```suggestion
			CLIEnvironmentSingleton().Logger().WarnfLn("--insecure-skip-tls-verify has no effect when --ca is set")
```
Also  `insecure-skip-tls-verify` and `ca` should be extracted to const